src/Runner.Common/JobServerQueue.cs

```diff
@@ -230,21 +232,37 @@ public void QueueFileUpload(Guid timelineId, Guid timelineRecordId, string type,
             _fileUploadQueue.Enqueue(newFile);
         }
 
-        public void QueueSummaryUpload(Guid stepRecordId, string name, string path, bool deleteSource)
+        public void QueueResultsUpload(Guid timelineRecordId, string name, string path, string type, bool deleteSource, bool finalize, bool firstBlock, long totalLines)
         {
+            if (!_resultsClientInitiated)
+            {
+                Trace.Verbose("Skipping results upload");
+                return;
+            }
+
+            if (timelineRecordId == _jobTimelineRecordId && String.Equals(type, CoreAttachmentType.ResultsLog, StringComparison.Ordinal))
+            {
+                Trace.Verbose("Skipping job log {0} for record {1}", path, timelineRecordId);
+                return;
```

Comment:
same here, do we need to delete the file?

Reply:
Not going to delete here since I am going to remove this check now in my next PR.  I already have the code mostly ready, just need to test and submit a PR.  

The general ff case is handled by the first delete above.